cppcache/integration-test-2/framework/Cluster.h

```diff
@@ -182,6 +182,8 @@ class Cluster {
         initialServers_(initialServers.get()) {
     jmxManagerPort_ = Framework::getAvailablePort();
 
+    // Clean up from previous run, if any, to ensure a clean server
```

Comment:
This comment is redundant to the statement below. Code should be expressive such that it doesn't need comments. Code needing preceding comments should be refactored to be more expressive.

Reply:
I would like to see an integration test written to test this change.
I would also like to see one that asserts that `GfshCommand::execute()` throws an exception when `gfsh` exits non-zero.